barretenberg/cpp/src/barretenberg/proof_system/op_queue/ecc_op_queue.hpp

```diff
@@ -38,6 +38,69 @@ class ECCOpQueue {
 
     Point get_accumulator() { return accumulator; }
 
+    /**
+     * @brief Prepend the information from the previous queue (used before accumulation/merge proof to be able to run
+     * circuit construction separately)
+     *
+     * @param previous
+     */
+    void prepend_previous_queue(const ECCOpQueue& previous)
+    {
+        // Allocate enough space
+        std::vector<ECCVMOperation> raw_ops_updated(raw_ops.size() + previous.raw_ops.size());
+        // Copy raw_ops
+        std::copy(previous.raw_ops.begin(), previous.raw_ops.end(), raw_ops_updated.begin());
+        std::copy(raw_ops.begin(),
```

Comment:
i stared at this line quite a bit, can you be more explicit in comments?

Reply:
Added a few